test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven/before/src/main/java/foo/bar/S3Streaming.java

```diff
@@ -90,4 +106,38 @@ void putObjectRequesterPaysSetter() {
 
         PutObjectRequest requestWithFalse = new PutObjectRequest("bucket", "key", "location").withRequesterPays(false);
     }
+
+    void putObjectRequest_setMetadata() {
+        ObjectMetadata metadata = new ObjectMetadata();
+        metadata.setContentLength(66);
+        metadata.setContentType("text/plain");
+        metadata.setContentEncoding("UTF-8");
+
+        PutObjectRequest request = new PutObjectRequest("bucket", "key", "location");
+        request.setMetadata(metadata);
+    }
+
+    void putObjectRequest_withMetadata() {
```

Comment:
Can we add a test with empty metadata, i.e., just `ObjectMetadata metadata = new ObjectMetadata();`